test/integration/testdata/gen-plugin-e2e-kube-flag-still-works.golden

```diff
@@ -1,3 +1,4 @@
+Flag --kube-conformance-image-version has been deprecated, Use --kubernetes-version instead.
```

Comment:
Hey! Good news this showed up; just noticed it on a self-review but we do want this shown. In sonobuoy gen it is unlikely they'll see it though.